pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -209,27 +192,6 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		/// Set a domain's router,
-		#[pallet::weight(T::WeightInfo::set_domain_router())]
-		#[pallet::call_index(0)]
-		pub fn set_domain_router(
```

Comment:
IIUC, this is replaced with `pallet_axelar_router::Pallet::<T>::set_config`?

Reply:
Correct!